workspace_host/interface.js

```diff
@@ -97,7 +98,7 @@ app.get(
     } else {
       res.status(200);
     }
-    res.jsonp({
+    res.json({
```

Comment:
AFAICT there's no legitimate reason to be using `jsonp` here, as this is strictly server-to-server communication.